tsconfig.json

```diff
@@ -5,7 +5,7 @@
     "baseUrl": "src",
     "outDir": "./dist/out-tsc",
     "forceConsistentCasingInFileNames": true,
-    "strict": true,
+    "strict": false,
```

Comment:
Why do you change this property?

Reply:
i dont want to give initial value for my variables